pages/services/prometheus/0.1.0-2.3.1/deploy-best-practice/index.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: layout.pug
+navigationTitle:  Deployment Best Practices
+title: Deployment Best Practices
+menuWeight: 30
+excerpt: Best practices for production deployment
+featureMaturity:
+enterprise: false
+---
+# Deployment Best Practices
+
+Note that the Node Exporter is best run on the host system, uncontainerized, so that it can cleanly expose all the host system metrics.
+
+## Configuration Best Practices for Production
+## Hardware Sizing Recommendations
+
+In practice this can only really be determined empirically, as it varies organisation by organisation and machine by machine.
+Below link will help deciding on how much ram does your prometheus would need :
+    
+    https://www.robustperception.io/how-much-ram-does-my-prometheus-need-for-ingestion/
+
+## Block Device / Storage
+
+Prometheus disk is hard to determing as it all depends on the organization to organization.
+Prometheus includes a local on-disk time series database, but also optionally integrates with remote storage systems.
```

Comment:
Has this been tested? If yes, the example must include the sample configuration

Reply:
This is not tested yet hence will not be able to give figures as informed this is all depends on organization to organization hence would not be easy to estimate.